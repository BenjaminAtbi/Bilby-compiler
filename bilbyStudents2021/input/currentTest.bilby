main {
<<<<<<< Updated upstream

  print true == false != false $n;
  print false == true != true $n;
  print true == true != false $n;
  print true == false != true $n;
  print false == true != false $n;
  print false == false != true $n;
  print true == true != true $n;
  print false == true != true $n;
  print true == false != true $n;
  print true == false != true $n;
  print true == false != true $n;
  print true == true != false $n;

  print false != false == false $n;
  print false != false == false $n;
  print false != false == false $n;
  print true != true == true $n;
  print false != false == true $n;
  print true != false == false $n;
  print false != true == false $n;
  print false != true == true $n;
  print false != true == true $n;
  print false != true == false $n;
  print true != true == true $n;
  print true != true == false $n;

  print false == true != true == true $n;
  print true == true != true == false $n;
  print true == true != false == false $n;
  print true == true != true == true $n;
  print true == false != true == true $n;
  print true == true != true == false $n;
  print true == false != false == false $n;
  print true == true != false == false $n;
  print false == false != false == false $n;
  print false == false != true == false $n;
  print true == true != false == false $n;
  print true == false != false == false $n;



}

=======
	mut c := alloc[[[int]]](3);
	c[1][1][1] := 2;
	mut d := 2;
	d := 1;
	mut val := c[1][1][1];
	print val $n;
	%c[0][0] := 3;
	%print c[0][0] $n;
}
>>>>>>> Stashed changes
