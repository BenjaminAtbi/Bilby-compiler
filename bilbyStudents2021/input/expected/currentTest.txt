[a, b, c, d, e]
[10, 7, 6]
[6.800000, 2.600000, 1.200000]
[fuck, shit, bitch]
[true, false, false, true]
